micro_brewery/demo/mrp_orderpoint_post.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
+    <data noupdate="1">
+        <record id="product_4_quant" model="stock.quant">
+            <field name="product_id" ref="product_product_4"/>
+            <field name="inventory_quantity">2000.00</field>
+            <field name="lot_id" ref="stock_lot_1"/>
+            <field name="location_id" model="stock.location" eval="obj().env.ref('stock.warehouse0').lot_stock_id.id"/>
+        </record>
+        <record id="product_5_quant" model="stock.quant">
+            <field name="product_id" ref="product_product_5"/>
+            <field name="inventory_quantity">1000.00</field>
+            <field name="lot_id" ref="stock_lot_3"/>
+            <field name="location_id" model="stock.location" eval="obj().env.ref('stock.warehouse0').lot_stock_id.id"/>
+        </record>
+        <record id="product_6_quant" model="stock.quant">
+            <field name="product_id" ref="product_product_6"/>
+            <field name="lot_id" ref="stock_lot_2"/>
+            <field name="inventory_quantity">1000.00</field>
+            <field name="location_id" model="stock.location" eval="obj().env.ref('stock.warehouse0').lot_stock_id.id"/>
+        </record>
+    </data>
+    <function name="action_apply_inventory" model="stock.quant">
+        <value eval="[
+                ref('product_4_quant'),
+                ref('product_5_quant'),
+                ref('product_6_quant')
+            ]"/>
+    </function>
+
+    <!-- orderpoint 00004 -->
+    <function name="write" model="stock.warehouse.orderpoint">
+        <value eval="[ref('stock_warehouse_orderpoint_4')]"/>
+        <value eval="{'qty_to_order': 86.00}"/>
+    </function>
+
+    <function name="action_replenish" model="stock.warehouse.orderpoint">
+        <value eval="[ref('stock_warehouse_orderpoint_4')]"/>
+    </function>
+
+    <!-- orderpoint 00006 -->
+    <function name="write" model="stock.warehouse.orderpoint">
+        <value eval="[ref('stock_warehouse_orderpoint_6')]"/>
+        <value eval="{'qty_to_order': 86.00}"/>
+    </function>
+
+    <function name="action_replenish" model="stock.warehouse.orderpoint">
+        <value eval="[ref('stock_warehouse_orderpoint_6')]"/>
+    </function>
+
+     <!-- orderpoint 00008 -->
+    <function name="write" model="stock.warehouse.orderpoint">
+        <value eval="[ref('stock_warehouse_orderpoint_8')]"/>
+        <value eval="{'qty_to_order': 60.00}"/>
+    </function>
+
+    <function name="action_replenish" model="stock.warehouse.orderpoint">
+        <value eval="[ref('stock_warehouse_orderpoint_8')]"/>
+    </function>
```

Comment:
Please split stock.quant, stock.warehouse.orderpoint and mrp.production into different files